comskip.c

```diff
@@ -700,7 +700,7 @@ unsigned int		min_black_frames_for_break = 1;
 bool				detectBlackFrames;
 bool				detectSceneChanges;
 int             dummy1;
-unsigned char*		frame_ptr;
+unsigned char*		frame_ptr = 0;

```

Comment:
Ah, is this the fix then?

I wonder why this is not being initialized to 0 for us.. AFAIK the compiler should be doing so for all globals.